nss-day-cohort-44/rare-rest-api-malicous-vortices
# Dd posts by title

# ISSUE TICKET 
## Description

Added the ability to search posts by title on server side

## Type of change


# Testing Instructions
`git fetch origin `
`git checkout ` 

Run a get request with this http://localhost:8000/posts?title=New and it should grab the posts in the seeded data.  You can replace what comes after = with whatever you want to search for.

User Instructions: 

# Checklist:
